src/main/resources/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerStep/config.jelly

```diff
@@ -38,7 +38,7 @@ THE SOFTWARE.
     <f:entry field="quietPeriod" title="Quiet period">
         <f:number clazz="number"/>
     </f:entry>
-    <f:entry title="Parameters">
+    <f:entry title="Parameters" help="${descriptor.getHelpFile('parameters')}">
```

Comment:
Does
```suggestion
    <f:entry title="Parameters" field="parameters">
```

suffice?

Reply:
Yeah, it looks like that works, added in f4d6f9b. I assumed `field` wasn't being used intentionally for some reason, but I tested and the snippet generator still seems to work correctly.